STX.REST.RESTFulSense.Clients/Brokers/Https/HttpBroker.Get.cs

```diff
@@ -0,0 +1,45 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace STX.REST.RESTFulSense.Clients.Brokers.Https
+{
+    internal partial class HttpBroker
+    {
+
+        public async ValueTask<T> GetContentAsync<T>(
+            string relativeUrl,
+            Func<string, ValueTask<T>> deserializationFunction = null)
+        {
+            HttpResponseMessage responseMessage =
+                await this.httpClient.GetAsync(relativeUrl);
+
+            return await DeserializeResponseContent<T>(
+                responseMessage, deserializationFunction);
+        }
+
+        public async ValueTask<T> GetContentAsync<T>(
+            string relativeUrl,
+            CancellationToken cancellationToken,
+            Func<string, ValueTask<T>> deserializationFunction = null)
+        {
+            HttpResponseMessage responseMessage =
```

Comment:
I think this method should not exist here because the deserialization/serialization is done under is own service.
Then, no sequencing allowed under brokers. No ifs, no iterations....